package.json

```diff
@@ -44,12 +44,14 @@
     "@adobe/alloy": "^2.4.0",
     "@adobe/react-spectrum": "^3.9.0",
     "@react-spectrum/combobox": "^3.0.0-beta.1",
+    "@react-stately/data": "^3.4.0",
     "@react/react-spectrum": "^2.34.0",
     "@spectrum-icons/illustrations": "^3.2.1",
     "@spectrum-icons/workflow": "^3.2.0",
-    "antd": "^4.12.3",
+    "antd": "^4.16.1",
```

Comment:
I upgraded to the latest ant because I needed to do some work to exclude ant's global style rules, which were messing up React-Spectrum's styling of validation icons within textfields (at least). https://github.com/ant-design/ant-design/issues/9363